tests/runtime/call

```diff
@@ -731,3 +731,16 @@ AFTER ./testprogs/syscall nanosleep  1e8
 NAME block expression with map assignment
 PROG BEGIN { @ = { let $a = 100; avg($a) }; exit(); }
 EXPECT @: 100
+
+NAME socket_cookie
```

Comment:
nit: I feel like these tests belong near those for `skboutput`, since they're both networking-related helpers.